### PR TITLE
fix grammar in user_lost_editor_role email

### PR DESCRIPTION
**Pull request by amengsk**
minor grammatical fix

### CODE REVIEW

**Review by tiltec — APPROVED** [review]
Good catch!

**Comment by nicksellen** [comment]
@amengsk you probably don't have merge rights right? I pressed "Update branch" button in GH, and will merge if the new build is successful :)

**Comment by amengsk** [comment]
thanks @nicksellen, indeed i do not have merge rights on any of the karrot repositories afaict

**Pull request merged**
